extensions/amp-social-share/amp-social-share.md

```diff
@@ -246,43 +262,15 @@ Sometimes you want to provide your own style. You can simply override the provid
 
 ```css
 amp-social-share[type='twitter'] {
+  color: blue;
```

Comment:
Related to this [comment thread](https://github.com/ampproject/amphtml/pull/31920/files#r556825448) - we could alternatively put disclaimer here in `### Custom Styles`?

Reply:
See above